chart/templates/clusterrole.yaml

```diff
@@ -26,8 +26,8 @@ rules:
   verbs: ["get", "list", "create", "update", "patch", "delete"]
 {{- end }}
 - apiGroups: ["federation.openshift-service-mesh.io"]
-  resources: ["meshfederations"]
+  resources: ["meshfederations", "federatedservices"]
   verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
 - apiGroups: ["federation.openshift-service-mesh.io"]
-  resources: ["meshfederations/status"]
+  resources: ["meshfederations/status", "federatedservices/status"]
```

Comment:
@jewertow I wonder how this was added. By running `make` I can't see those changes, so I assume you did it by hand. 

Created #138 to follow-up on this.